content/accommodations.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Accommodations"
+date: 2022-10-15T18:30:00+02:00
+weight: 10
+draft: false
+menu: main
+type: page
+---
+There are many hotels in Delft. Here is a list with possible hotels that aren’t really
+expensive and are easy to travel from to the campus.
+* The student hotel Delft
+  * Prize: 60-95 euros per night per person
+  * Breakfast possible for 25 euros
+  * 20 minutes walk or 5 minutes with bus to contest location
+  * Next to Delft station: Van Leeuwenhoekpark 1, 2611 DW, Delft
+  * More information: [https://www.thestudenthotel.com/delft/](https://www.thestudenthotel.com/delft/)
+* Hotel Royal Bridges
+  * Prize: 60-110 euros per night per person
+  * Breakfast possible for 15 euros
+  * 20 minutes walk or 5 minutes with bus to contest location
+  * In centrum: Koornmarkt 55-65, 2611 EC Delft
+  * More information: [https://www.royalbridges.nl/en/](https://www.royalbridges.nl/en/)
+* Ibis Styles Delft City Center
+  * Prize: 65-85 euros per night per person
```

Comment:
```suggestion
  * Cost: 65-85 euros per night per person
```
But if you prefer "price" over "cost", please write it with a "c" :stuck_out_tongue: